config/locales/forms/fr.yml

```
@@ -140,8 +140,10 @@ fr:
         déverouillage facial pour accéder à votre compte avec %{app_name}.
       instructions_title: Utilisez le déverrouillage par empreinte digitale ou le
         déverouillage facial pour accéder à votre compte.
-      intro: Utilisez le déverrouillage par empreinte digitale ou le déverouillage
-        facial pour accéder à votre compte.
+      intro: Activez le déverrouillage facial ou tactile comme méthode
+        d’authentification sur cet appareil. Vous aurez besoin de cet appareil
+        pour utiliser cette méthode à l’avenir. Commencez par donner un surnom à
+        votre appareil. facial pour accéder à votre compte.
```

Is the last part of this expected to be there?

```suggestion
        votre appareil.
```